rt/src/test/java/io/fabric8/maven/rt/BaseBoosterIT.java

```diff
@@ -192,6 +192,10 @@ protected Repository setupSampleTestRepository(String repositoryUrl, String rela
         return repository;
     }
 
+    protected String getCurrentFMPVersion() throws Exception{
```

Comment:
Actually in this case also we would end up querying project model again and again for fmp's version. I was talking about having it as a member element, like `testsuiteNamespace`